packages/terra-form-field/docs/MigrationGuide.md

```diff
@@ -0,0 +1,76 @@
+# Migrating from terra-form's Field Component to terra-form-field
+
+The following sections describe how to migrate from the Field component provided in the terra-form package to the Field component provided in the terra-form-field package. Terra-form-field provides enhanced API and layout to begin the first phase of the OSC uplift.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
```

Comment:
Do you need to mention getting started twice? Looks like it's specified in your README.md doc